Docker/README.md

```diff
@@ -0,0 +1,150 @@
+### Introduction
```

Comment:
@huitseeker if you could please do a first pass on this README it would be great. Talked with @Clay-Mysten and he'll try to have a look as well later this week